widgets/src/components/TaskList/TaskListContainer.js

```diff
@@ -191,9 +208,8 @@ class TaskList extends React.Component {
     return (
       <ThemeProvider theme={theme}>
         <Paper className={classes.paper}>
-          {error ? (
-            <ErrorComponent message={i18next.t('messages.errors.dataLoading')} />
-          ) : (
+          {blocker && <ErrorComponent message={blocker} />}
```

Comment:
the syntax:
```
{ blocker ? <ErrorComponent message={blocker} /> : (table) }
```
will be better in this case

Reply:
I agree. I have refactored it